rdr_service/participant_enums.py

```diff
@@ -35,7 +35,7 @@
 TEST_EMAIL_PATTERN = "%@example.com"
 # The name of the 'test' HPO that test participants are normally affiliated with.
 TEST_HPO_NAME = "TEST"
-TEST_HPO_ID = 19
+TEST_HPO_ID = 21
```

Comment:
It might be better to load the HPO by the name, just in case this gets out of date again.

Reply:
I wish I didn't have to keep adding more joins and/or queries (through the HPODao) to the generator logic vs. filtering on the `participant` table data that's already part of the query.  Is there a legitimate risk this will change again?

Reply:
I'm not sure how it changed the first time. But if it does change for some reason we're not likely going to think to update this code. And a random HPO's participants would be flagged as test participants.

Reply:
I changed things to use results from HPO table.